addons/web/static/src/core/file_input/file_input.js

```diff
@@ -54,6 +54,19 @@ export class FileInput extends Component {
     }
 
     async uploadFiles(params) {
+        if ((params.ufile && params.ufile.length) || params.file) {
+            const fileSize = (params.ufile && params.ufile[0].size) || params.file.size;
+            if (!checkFileSize(fileSize, this.env.services.notification)) {
+                // FIXME
+                // Note that the notification service is not added as a
+                // dependency of this component (through useService hook),
+                // in order to avoid introducing a breaking change in a stable version.
+                // If the notification service is not available, the
+                // checkFileSize function will not display any notification
+                // but will still return the correct value.
+                return null;
+            }
+        }
```

Comment:
It seems the code has changed in that version, so maybe we have something else to do. I let you check with your team. Also, I will let your team do the r+ of that forward port when it is good for you. 🙂

Reply:
It's because the forward port for the PR which introduces the issue still hadn't been merged: https://github.com/odoo/odoo/pull/143114
I will rebase now and it should be good